lib/clever/types/student.rb

```diff
@@ -6,17 +6,19 @@ class Student < Base
       attr_reader :uid,
                   :first_name,
                   :last_name,
-                  :provider
+                  :provider,
+                  :legacy_id
 
       def initialize(attributes = {}, client: nil)
-        @uid               = attributes['id']
+        @district_username = attributes.dig('credentials', 'district_username')
+        @email             = attributes['email']
         @first_name        = attributes['name']['first']
         @last_name         = attributes['name']['last']
-        @district_username = attributes.dig('credentials', 'district_username')
+        @legacy_id         = attributes.dig('roles', 'student', 'legacy_id')
```

Comment:
Students don't have legacy ids as far as I've seen, but adding it here anyway to give us flexibility on calling `legacy_id` on a student or a user record in Plato and not have it break